packages/ui/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@condo/ui",
+  "description": "A set of React UI components for developing applications inside the condo ecosystem",
+  "version": "0.0.1",
+  "repository": "https://github.com/open-condo-software/condo/blob/master/packages/ui",
+  "author": {
+    "name": "Matthew Savelev",
+    "email": "msavelyev@doma.ai",
+    "url": "https://github.com/SavelevMatthew"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prepare:tokens": "rm -rf src/tokens/sets && node bin/prepare-tokens.js",
+    "build:tokens": "find src/tokens -name 'variables.*' -delete && node bin/build-vars.js",
```

Comment:
it looks like a part of `prepare:tokens` command

Reply:
Sometimes we don't want to generate tokens all the times. But maybe you're right...